fuel-core/src/cli/run.rs

```diff
@@ -71,9 +75,18 @@ impl Command {
             min_gas_price,
             predicates,
             relayer_args,
+            p2p_args,
         } = self;
 
         let addr = net::SocketAddr::new(ip, port);
+
+        let p2p = {
+            match p2p_args.into() {
+                Ok(value) => value,
+                Err(e) => return Err(io::Error::new(io::ErrorKind::Other, e)),
```

Comment:
Any particular reason for `io::Error` over something like `anyhow::Error`?

Reply:
mainly because the return value of the function is `io::Result` not `anyhow::Result`